parmesan/__init__.py

```diff
@@ -9,7 +9,7 @@
 
 Please make sure you install a recent enough version of Theano.  See
 section 'Install from PyPI' in the installation docs for more details:
-http://parmasan.readthedocs.org/en/latest/user/installation.html#install-from-pypi
+http://parmesan.readthedocs.org/en/latest/user/installation.html#install-from-pypi
```

Comment:
It should probably refer to lasagne? We are not using readthe docs and the note is there because we copied lasagnes setup.